lib/identity_config.rb

```diff
@@ -202,6 +202,7 @@ def self.build_store(config_map)
     config.add(:phone_setups_per_ip_period, type: :integer)
     config.add(:pii_lock_timeout_in_minutes, type: :integer)
     config.add(:pinpoint_sms_configs, type: :json)
+    config.add(:pinpoint_sms_sender_id, type: :string, allow_nil: true)
```

Comment:
Just to confirm: Sender IDs aren't distinct per country?

Reply:
The way it works is we create a sender ID and register it with various countries. So we *could* have per-country sender IDs but as current configured, we point multiple countries at our same sender ID

https://gsa-tts.slack.com/archives/CR2SSAFRQ/p1625750846003400